src/features/locators/reducers/runLocatorsGeneration.thunk.ts

```diff
@@ -25,17 +25,24 @@ export const runLocatorsGeneration = createAsyncThunk(
   "locators/runLocatorsGeneration",
   async (meta: Meta, thunkAPI) => {
     const state = thunkAPI.getState() as RootState;
-
+    const currentPageObjLibrary = selectLastElementLibrary(state);
```

Comment:
careful! last PO isn't necessary the current. use another selector to get current PO

Reply:
thanks!